docs/index.md

```diff
@@ -76,58 +76,107 @@ connection "alicloud" {
+## Multi-Account Connections
 
-### Specify multiple accounts 
-A common configuration is to have multiple connections to different accounts:
+You may create multiple alicloud connections:
 
 ```hcl
-connection "ali_account_aaa" {
+connection "alicloud_dev" {
   plugin      = "alicloud"
   secret_key  = "gMCYsoGqjfThisISNotARealKeyVVhh"
   access_key  = "ASIA42DZSWFYSN2PFHPJ"
-  regions     = ["us-east-1" , "ap-south-1"]
+  regions     = ["eu-central-1" , "cn-hangzhou"]
+}
+
+connection "alicloud_qa" {
+  plugin      = "alicloud"
+  secret_key  = "gMCYsoGqjfThisAintARealKeyVVhh"
+  access_key  = "ASIA42DZSWFYS42PFJHP"
+  regions     = ["cn-hangzhou"]
 }
 
-connection "ali_account_bbb" {
+connection "alicloud_prod" {
   plugin      = "alicloud"
   secret_key  = "gMCYsoGqjfThisAintARealKeyVVhh"
   access_key  = "ASIA42DZSWFYS42PFJHP"
-  regions     = ["cn-east-1"]
+  regions     = ["cn-hangzhou"]
 }
+```
+
+Each connection is implemented as a distinct [Postgres schema](https://www.postgresql.org/docs/current/ddl-schemas.html). As such, you can use qualified table names to query a specific connection:
+
+```sql
+select * from alicloud_qa.alicloud_account;
+```
+
+You can multi-account connections by using an [**aggregator** connection](https://steampipe.io/docs/using-steampipe/managing-connections#using-aggregators). Aggregators allow you to query data from multiple connections for a plugin as if they are a single connection:
 
+```hcl
+connection "alicloud_all" {
+  plugin      = "alicloud"
+  type        = "aggregator"
+  connections = ["alicloud_dev", "alicloud_qa", "alicloud_prod"]
+}
 ```
 
-### Specify static credentials using environment variables 
+Querying tables from this connection will return results from the `alicloud_dev`, `alicloud_qa`, and `alicloud_prod` connections:
+
+```sql
+select * from alicloud_all.alicloud_account;
+```
+
+Alternatively, can use an unqualified name and it will be resolved according to the [Search Path](https://steampipe.io/docs/guides/search-path). It's a good idea to name your aggregator first alphbetically, so that it is the first connection in the search path (i.e. `alicloud_all` comes before `alicloud_dev`):
+
+```sql
+select * from alicloud_account;
+```
+
+Steampipe supports the `*` wildcard in the connection names. For example, to aggregate all the Alicloud plugin connections whose names begin with `alicloud_`:
+
+```hcl
+connection "alicloud_all" {
+  type        = "aggregator"
+  plugin      = "alicloud"
+  connections = ["alicloud_*"]
+}
+```
+
+Aggregators are powerful, but they are not infinitely scalable. Like any other Steampipe connection, they query APIs and are subject to API limits and throttling. Consider as an example and aggregator that includes 3 Alicloud connections, where each connection queries 28 regions. This means you essentially run the same list API calls 84 times! When using aggregators, it is especially important to:
```

Comment:
I think Alicloud supports more than 28 regions - https://www.alibabacloud.com/help/en/basics-for-beginners/latest/regions-and-zones